ui-library-pstn-1-n-calling/src/App.tsx

```diff
@@ -1,123 +1,59 @@
   useAzureCommunicationCallAdapter,
 } from "@azure/communication-react";
-import React, { CSSProperties, useMemo, useRef } from "react";
-import { v4 as uuidv4 } from "uuid";
-
-/**
- * Authentication information needed for your client application to use
- * Azure Communication Services.
- *
- * For this quickstart, you can obtain these from the Azure portal as described here:
- * https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/identity/quick-create-identity
- *
- * In a real application, your backend service would provide these to the client
- * application after the user goes through your authentication flow.
- */
-const USER_ID = "<Azure Communication Services Identifier>";
-const TOKEN = "<Azure Communication Services Access Token>";
-
-/**
- * Display name for the local participant.
- * In a real application, this would be part of the user data that your
- * backend services provides to the client application after the user
- * goes through your authentication flow.
- */
-const DISPLAY_NAME = "<Display Name>";
+import React, { useMemo } from "react";
+import { INPUTS } from "./INPUTS";
 
 /**
  * Entry point of your application.
  */
 function App(): JSX.Element {
-  // Arguments that would usually be provided by your backend service or
-  // (indirectly) by the user.
-  const { userId, token, displayName, groupId } =
-    useAzureCommunicationServiceArgs();
-
-  // A well-formed token is required to initialize the calling adapter.
-  const credential = useMemo(() => {
-    try {
-      return new AzureCommunicationTokenCredential(token);
-    } catch {
-      console.error("Failed to construct token credential");
-      return undefined;
-    }
-  }, [token]);
-
   // Memoize arguments to `useAzureCommunicationCallAdapter` so that
   // a new adapter is only created when an argument changes.
   const callAdapterArgs = useMemo(
-    () => ({
-      userId: fromFlatCommunicationIdentifier(
-        userId
-      ) as CommunicationUserIdentifier,
-      displayName,
-      credential,
-      locator: {
-        // Can be a phone number if you want to call a PSTN number. For example, "+18001234567"
-        // alternateCallerId is required for calling phone numbers using the call composite.
-        // NOTE: Feature in public preview. Not recommended for production use.
-        groupId,
-      },
-      // For using PSTN Calling features in CallComposite.
-      // Read more about procuring a phone number from Azure
-      // https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/telephony/get-phone-number
-      // NOTE: Feature in public preview. Not recommended for production use.
-      alternateCallerId: "<Azure Communication Services provided Phone Number>",
-    }),
-    [userId, credential, displayName, groupId]
+    () => {
+      try {
+        return {
+          userId: fromFlatCommunicationIdentifier(INPUTS.userIdentity) as CommunicationUserIdentifier,
```

Comment:
nit.. perhaps we can use the spread operator to output to the various variables instead of adding INPUTS. before each one?

Reply:
Updated to spread 👍